src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -129,6 +129,7 @@ class StockMovement {
             name                : name,
             description         : description,
             statusCode          : statusCode,
+            statusLabel         : RequisitionStatus.getRequestStatusLabel(requisition?.status, shipment?.status),
```

Comment:
If we end up keeping this, let's call this displayStatus (we use this in other places like Order and Shipment). 

FWIW, we already have displayStatusCode in StockMovementStatusCode enum as well. But I think this is used to "mask" the actual status from certain users. For example, whether a request is in REQUESTED or PICKING doesn't matter to the requestor ... the request is in a PENDING state. If we need to display the status to the user, then it seems we just need to add all of the approval workflow states to StockMovementStatusCode along with the appropriate displayStatusCode which is just the same status.
```
PENDING_APPROVAL(2, PENDING_APPROVAL),
APPROVED(2, APPROVED),
REJECTED(2, REJECTED),
```

Reply:
Lastly, in order to uncomplicate things (i.e. having requisition status and shipment status), I don't think we need to show SHIPPED as a status. For a requisition, it's just ISSUED.